### PR TITLE
kubeadm: add note about a bug in the PublicKeysECDSA feature gate

### PR DESCRIPTION
**Pull request by SequoiaGod**
Add note about that in the FG section of the kubeadm init doc.
xref https://github.com/kubernetes/website/issues/46799

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -175,9 +175,8 @@ as a learner and promoted to a voting member only after the etcd data are fully
 Renewal of existing ECDSA certificates is also supported using `kubeadm certs renew`, but you cannot
 switch between the RSA and ECDSA algorithms on the fly or during upgrades. Kubernetes
 {{< skew currentVersion >}} has a bug where keys in generated kubeconfig files are set use RSA
-despite the feature gate being enabled.
-: Kubernetes versions before v1.31 had a bug where keys in generated kubeconfig files were set use RSA,
-even when you had enabled the `PublicKeysECDSA` feature gate.
+despite the feature gate being enabled. Kubernetes versions before v1.31 had a bug where keys in generated kubeconfig files
+were set use RSA, even when you had enabled the `PublicKeysECDSA` feature gate.
```
Comment (tengqm) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md:179]: Can we combine this sentence with the previous one to make it clearer?

**Review by neolit123 — COMMENTED** [review]
thanks!
/lgtm
/sig cluster-lifecycle
/label tide/merge-method-squash

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md:179]: - Happy to see a further PR that tweaks the wording.
- It's really helpful to document the bug
- We'll need to resolve this one way or another when merging _main_ into _dev-1.31_ anyway.

**Pull request merged**